src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/out/msg/EnvelopeMsg.java

```diff
@@ -0,0 +1,48 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.model.out.msg;
+
+import com.google.common.base.Strings;
+
+import java.nio.charset.StandardCharsets;
+
+public class EnvelopeMsg implements Msg {
+
+    private final String envelopeId;
+
+    private final byte[] envelope;
+
+
+    public EnvelopeMsg(String envelopeId) {
+        this.envelopeId = envelopeId;
+        this.envelope = new byte[0];
+    }
```

Comment:
I am afraid this will have to stay as it is because otherwise it fails code coverage and another reviewer will block the merge. More than writing code I am trying to make you all happy guys ;-)

Reply:
Didn't see/have latest changes when commented. All previous constructors are gone so my _note_ is invalid 😉 . NW @luigibk , we all like writing code and want others to be happy